bot/cogs/giveaway.py

```diff
@@ -0,0 +1,171 @@
+import logging
+from datetime import datetime, timedelta, timezone
+from typing import Optional
+
+import discord
+from discord import app_commands, Interaction
+from discord.ext import commands, tasks
+
+from bot.bot import WarnetBot
+from bot.config import GiveawayConfig, GUILD_ID
+
+logger = logging.getLogger(__name__)
+
+
+@commands.guild_only()
+class Giveaway(commands.GroupCog, group_name='warnet-ga'):
+    def __init__(self, bot: WarnetBot) -> None:
+        self.bot = bot
+        self.db_pool = bot.get_db_pool()
+
+    @commands.Cog.listener()
+    async def on_connect(self) -> None:
+        self._check_blacklist_ga.start()
+
+    @app_commands.command(name='blacklist', description='Blacklist a user from giveaway')
+    @app_commands.describe(
+        amount='Giveaway prizes. Example: 50000',
+        winner='Winner of the giveaway. Example: 1234567890,0987654321',
+        ghosting='Member who do not claim the giveaway. Example: 1234567890,0987654321',
```

Comment:
The example is too long for a parameter description

Reply:
I don't think it's too long, it helps users understand how to separate them if more than one is entered